advisor/src/main/kotlin/VulnerabilityProvider.kt

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2020-2021 Bosch.IO GmbH
+ * Copyright (C) 2021 HERE Europe B.V.
```

Comment:
Not sure if this copyright is needed.

Reply:
My reasoning is that I copied code from the (formerly) [Advisor](https://github.com/oss-review-toolkit/ort/blob/d70e7cf5e096ab3ec368d5309d8c473a5284b994/advisor/src/main/kotlin/Advisor.kt#L84-L116) which has this copyright.